### PR TITLE
Fix opf_checkpoint_test.py

### LINKED ISSUE #2799: Fix opf_checkpoint_test.py
The two enabled tests in `opf_checkpoint_test.py` are invalid after the first time they are run because they do not clean up the checkpoint directories they produce, and they reuse these directories on subsequent runs.

### PR DESCRIPTION
**Pull request by chetan51**
Fixes #2799.

### CODE REVIEW

**Comment by chetan51** [comment]
@scottpurdy

**Comment by rhyolight** [comment]
:+1: @scottpurdy ?

File: tests/integration/nupic/opf/opf_checkpoint_test/opf_checkpoint_test.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 import shutil
 
 from nupic.data.file_record_stream import FileRecordStream
-from nupic.frameworks.opf.experiment_runner import runExperiment
+from nupic.frameworks.opf.experiment_runner import runExperiment, _getCheckpointParentDir
```
Comment (scottpurdy) [inline tests/integration/nupic/opf/opf_checkpoint_test/opf_checkpoint_test.py]: Can we change `_getCheckpointParentDir` to be a public function explicitly? It seems like something that clients of experiment runner may wish to use.

**Comment by scottpurdy** [comment]
:+1:

**Pull request merged**